calendar/meet-calendar.js

```diff
@@ -784,7 +784,7 @@ class G2Description extends Description {
      * @returns {*}
      */
     get element() {
-        var description = $('#xDescIn > [role="textbox"]');
+        var description = $('#xDescIn > [contenteditable="true"]');
```

Comment:
xDescIn is not (no longer?) editable using the textbox attribute. This fixes it by going straight to the child element that is editable.